src/store/flow/block.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from '@floip/flow-runner'
 import { ActionTree, GetterTree, MutationTree } from 'vuex'
 import { IRootState } from '@/store'
-import { defaults, without } from 'lodash'
+import { defaults, set } from 'lodash'
```

Comment:
Noticed you also do this in 2296. I suppose this will shake out in the conflict resolution...

Reply:
Thanks for noticing. I think it will be a conflict. Anyway, any change in this line would create a conflict, so I will leave this here.